components/mdc/TextInput/helpers.js

```diff
@@ -1,3 +1,9 @@
 export const addOrRemoveInvalidClass = async (isInvalid, element) => {
   isInvalid ? element.classList?.add('mdc-text-field--invalid') : element.classList?.remove('mdc-text-field--invalid')
 }
+
+export function getDecimalPlacesLength(number = 0) {
```

Comment:
Why the default value on this parameter? (It doesn't seem like the sort of function we'd call without providing any argument.)

Reply:
true, this only protects against step being set to '' which is possible in storybook, so consumers don't really need it since step has a default value, but I could also leave it there for storybooks sake.

Reply:
I think the default value is actually unnecessary even in that situation, because the default will only be used for a call like this (with no argument provided)...
```
getDecimalPlacesLength()
```
... not a call like this (regardless of what value `step` has):
```
getDecimalPlacesLength(step)
```

Reply:
if step is an empty string Storybook throws an error without the default value since "" is falsey.

Reply:
we could do something like `if(!number) number = 0` but that seems to be what js is doing for us already.